.github/workflows/ci-cd.yml

```
@@ -41,20 +41,15 @@ jobs:
         os:
         - ubuntu-24.04
         - ubuntu-24.04-arm
```

Could be useful to be more explicit:
```suggestion
        - ubuntu-24.04-arm
        experimental:
        - false
```

---

I don't think this makes sense as a change - there's no world in which we want to parametrize an entry to run with both experimental as `true` and `false`, so it does not belong in the matrix.

---

I tried to follow [GitHub's documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations) and it obviously works. Feel free to structure the matrix differently, if you want to apply "best practices". I have no strong opinion either way.

---

Yep, this way is what's suggested in GitHub's own docs.